docs/core/install/linux.md

```diff
@@ -14,55 +14,31 @@ ms.date: 11/08/2022
 > - [Install on macOS](macos.md)
 > - [Install on Linux](linux.md)
 
-This article details how to install .NET on various Linux distributions, manually and via a package manager. Typically, stable .NET versions are available in a package manager, and Preview versions are not.
-
-The latest version of .NET is 7.
+This article details how to install .NET on various Linux distributions either manually, via a package manager, or via a [container](../docker/introduction.md#net-core-images).
 
 ## Manual installation
 
 You can install .NET manually in the following ways:
 
-- [Download tarballs](https://dotnet.microsoft.com/download/dotnet)
+- [Manual install](linux-scripted-manual.md#manual-install)
 - [Scripted install](linux-scripted-manual.md#scripted-install)
-- [Manual binary extraction](linux-scripted-manual.md#manual-install)
 
 You may need to install [.NET dependencies](https://github.com/dotnet/core/blob/main/release-notes/7.0/linux-packages.md) if you install .NET manually.
 
-## Official package archives
-
-.NET is available in the [official package archives](https://pkgs.org/search/?q=dotnet) for various Linux distributions, including the following ones:
-
-- [Alpine Linux](https://pkgs.alpinelinux.org/packages?name=dotnet*)
-- [Arch Linux](https://archlinux.org/packages/?q=dotnet)
-- [Arch Linux User Repository](https://aur.archlinux.org/packages?K=dotnet)
-- [Fedora](https://packages.fedoraproject.org/search?query=dotnet)
-- [Red Hat Enterprise Linux](https://access.redhat.com/documentation/en-us/net/6.0)
-- [Ubuntu](https://packages.ubuntu.com/search?keywords=dotnet6)
-
-Microsoft collaborates with partners to ensure .NET works well on their Linux distributions. Support is provided by those distributions. You can still [open issues at dotnet/core](https://github.com/dotnet/core/issues) if you run into problems.
+## Packages
 
-## Microsoft packages
-
-.NET is also available via [packages.microsoft.com](https://packages.microsoft.com/).
+.NET is available in [official package archives](https://github.com/dotnet/core/blob/main/linux.md) for various Linux distributions and [packages.microsoft.com](https://packages.microsoft.com/).
 
+- [Alpine](linux-alpine.md)
```

Comment:
We need instructions added to #32596 for Alpine.

Reply:
We can add that in a separate PR.